DEV.md

```diff
@@ -0,0 +1 @@
+Using workspaces: https://classic.yarnpkg.com/en/docs/workspaces/
```

Comment:
Let's name this file `CONTRIBUTING.md` following the common practices.

Reply:
++